encoders/multimodal/TirgMultimodalEncoder/__init__.py

```diff
@@ -64,7 +64,7 @@ def _get_features(self, data):
 
         return self.model.compose_img_text_features(img_features, text_features)
 
-    @batching_multi_input(num_data=2)
+    @batching(batch_size=2)
```

Comment:
is not batch size, is slice_nargs

Reply:
This reminds me of how to config batch_size for these encoders? It seems that we don't expose this param at the moment? @JoanFM